src/tests/ftest/vmd/fault_reintegration.py

```diff
@@ -0,0 +1,234 @@
+            "plugin_path": None,
+            "dfuse": None,
+            "display_space": True,
+            "fail_on_warning": False,
+            "namespace": "/run/ior/*",
+            "ior_params": {
+                "oclass": oclass,
+                "flags": self.ior_w_flags,
+                "transfer_size": ior_test_seq[2],
+                "block_size": ior_test_seq[3]
+            }
+        }
+        kwargs.update(ior_kwargs)
+        thread.append(threading.Thread(target=thread_run_ior, kwargs=kwargs))
+
+        # Launch the IOR thread
+        self.log.info("Start IOR thread : %s", thread)
+        thread[0].start()
+
+        # 4. While IOR (EC+GX) in progress put single drive to fault mode
+        self.log_step("dmg storage set nvme-faulty and check led state")
+        if not self.check_result(self.set_device_faulty(test_dev), "EVICTED", "ON"):
+            self.fail("#Result of set_device_fault, device not in expected EVICTED, ON state")
+        # check device state after set nvme-faulty
+        time.sleep(1)
+        if not self.verify_dev_led_state(test_dev, "NORMAL", "ON"):
+            self.fail("#After set_device_fault, device not back to NORMAL, ON state")
+
+        # 5. IOR will halt until rebuild is finished.
+        self.log_step("IOR continue")
+        # 6. Wait until the IOR thread completed and check for error
+        self.log_step("IOR completed and check for error")
+        thread[0].join()
+        errors = 0
+        while not thread_queue.empty():
+            result = thread_queue.get()
+            self.log.debug("Results from thread %s (log %s)", result["job_id"],
+                           result["log"])
+            for name in ("command", "exit_status", "interrupted", "duration"):
+                self.log.debug("  %s: %s", name, getattr(result["result"], name))
+            for name in ("stdout", "stderr"):
+                self.log.debug("  %s:", name)
+                for line in getattr(result["result"], name).splitlines():
+                    self.log.debug("    %s:", line)
+            if result["result"].exit_status != 0:
+                errors += 1
+        if errors:
+            self.fail("Errors running IOR {} thread".format(errors))
+
+        # 7. Check drive status LED status
+        self.log_step("Check drive status and led status")
+        if not self.verify_dev_led_state(test_dev, "NORMAL", "ON"):
+            self.fail(
+                "#After set_device_fault, IOR completed, device not in expected NORMAL, ON state")
+
+        # Verify IOR data and check container
+        self.log.info("Verify IOR data")
+        ior_kwargs["ior_params"]["flags"] = self.ior_r_flags
+        ior_kwargs["log"] = "ior_read_pool_test.log"
+        try:
+            run_ior(**ior_kwargs)
+        except CommandFailure as error:
+            self.fail("Error in ior read {}.".format(error))
+        kwargs = {"pool": self.pool.uuid,
+                  "cont": self.container.uuid}
```

Comment:
We should use labels when possible
```suggestion
        kwargs = {"pool": self.pool.identifier,
                  "cont": self.container.identifier}
```